task/tekton-prune/0.1/tekton-prune.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: tekton-prune
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: tekton, misc
+    tekton.dev/displayName: "tekton objects cleanup"
```

Comment:
I guess `tekton prune` would look great in the main menu where all the tasks are listed :sweat_smile: